source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
```

Comment:
@backflip just tried it in current project (didn't have a look in depth) without success. I think that removing this needs more check, so we probably need to test it in a clean project first.